### ISSUE TITLE
Digest qop different than "auth" or "auth-int", authentication aborted

### ISSUE THREAD

**Issue by prozaklob** [flutter-webrtc/dart-sip-ua#151]
hi!
(possibly after plugins upgrade) Registration has ceased.
dart-sip-ua 0.3.1
webrtc  0.4.0

with:
 sip_ua: "0.2.4"
 flutter_webrtc: "0.2.8"
work fine

```
I/flutter (24824): [2020-11-04 21:39:00.582] Level.error digest_authentication.dart:132 ::: authenticate() | challenge without Digest qop different than "auth" or "auth-int", authentication aborted
I/flutter (24824): [2020-11-04 21:39:00.591] Level.debug sip_ua_helper.dart:146 ::: registrationFailed => Code: [401], Cause: Authentication Error, Reason: Unauthorized
```

```
[√] Flutter (Channel stable, 1.22.3, on Microsoft Windows [Version 10.0.18363.1139], locale ru-RU)
    • Flutter version 1.22.3 at C:\Users\admin\Documents\flutter
    • Framework revision 8874f21e79 (6 days ago), 2020-10-29 14:14:35 -0700
    • Engine revision a1440ca392
    • Dart version 2.10.3


[√] Android toolchain - develop for Android devices (Android SDK version 30.0.2)
    • Android SDK at d:\android\sdk_platform\
    • Platform android-30, build-tools 30.0.2
    • ANDROID_SDK_ROOT = D:\android
    • Java binary at: D:\android\sdk\jre\bin\java
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b01)
    • All Android licenses accepted.

[√] Android Studio (version 4.0)
    • Android Studio at D:\android\sdk
    • Flutter plugin version 50.0.1
    • Dart plugin version 193.7547
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b01)

[√] VS Code (version 1.50.1)
    • VS Code at C:\Users\admin\AppData\Local\Programs\Microsoft VS Code
    • Flutter extension version 3.16.0

[√] Connected device (1 available)
    • ONEPLUS A5000 (mobile) • 20a6b3b • android-arm64 • Android 10 (API 29)

• No issues found!
```

**Comment by cloudwebrtc** [comment]
Can you provide relevant SIP logs?

**Comment by cloudwebrtc** [comment]
Fixed in 0.3.2
